src/components/search-query.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import PropTypes from 'prop-types';
+
+import { RefinementList, InstantSearch, SearchBox, Hits, ClearRefinements, Pagination } from 'react-instantsearch-dom';
+import webComponent from '../web-component';
+
+const searchClient = algoliasearch('H897LKXYG1', '50a4f124e0d934cac92e79ece376316a');
+
+const hitComponent = ({ hit }) => (
+  <div>
+    { hit.last_name ?
+      <div><strong>Contributor:</strong> {hit.first_name} {hit.last_name}</div> : ''
+    }
```

Comment:
I'm a stickler for consistent code styling. It may seem tedious or unnecessary, but it can help _a lot_ to maintain clean code and helps keep a better experience for the other folks on the team who may be working on the same files. Ideally, we'd have some sort of "format-on-save" functionality for the project, but until then, we'll just need to be extra-attentive to the structure of the code.

```suggestion
    {hit.last_name ?
      <div>
        <strong>Contributor: </strong>{hit.first_name} {hit.last_name}
      </div>
      : ''
    }
```

**Note:** These suggestions don't actually change the content of the code, they just edit a bit of styling and spacing of the code, so that each element looks as close to the other elements as possible.

Reply:
I agree coding style is important.  Takes some time to get used to a new one.

Reply:
I agree code styling is important.  Takes a while to get used to a new one.

Reply:
Sure thing! This was a great start, my suggestions are just to tidy up the html a bit, but overall it's been fine :)